hadoop-hdfs-project/hadoop-hdfs/src/main/java/org/apache/hadoop/hdfs/qjournal/server/JournalNodeSyncer.java

```diff
@@ -153,7 +156,8 @@ private boolean getOtherJournalNodeProxies() {
         LOG.warn("Could not add proxy for Journal at addresss " + addr, e);
       }
     }
-    if (otherJNProxies.isEmpty()) {
+    // Check if any of there are any resolvable JournalNodes before starting the sync.
```

Comment:
s/there/them/? Extra 'any'?

Reply:
I've realized that not starting the syncer because the addresses are unresolved is counter-productive.  The act of attempting to sync instigates attempts to communicate with the other servers, which will result in resolution if they are available.  I'll restart the original check for "are there any other configured proxies".